src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/CrossTarget/CrossTargetRuleSubscriberBase.cs

```diff
@@ -243,10 +253,18 @@ private IEnumerable<string> GetWatchedRules(RuleHandlerType handlerType)
                                                   isActiveContext,
                                                   ruleChangeContext)
                                      .ConfigureAwait(true);
+
+                        // check full set of handler rules for changes
+                        _treeTelemetryService.ObserveHandlerRulesChanges(
+                            projectContextToUpdate.TargetFramework,
+                            handler.GetRuleNames(RuleHandlerType.DesignTimeBuild), 
```

Comment:
>DesignTimeBuild [](start = 65, length = 15)

What if handler does not support DTBuild but evaluation only? In this case you would either not send event at all , or send it as Unresolved, when it could be resolved since other handlers that do support DTBuild di resolve.